client/components/domains/register-domain-step/index.jsx

```diff
@@ -407,7 +407,7 @@ const RegisterDomainStep = React.createClass( {
 						cart={ this.props.cart }
 						selectedSite={ this.props.selectedSite }
 						domainsWithPlansOnly={ this.props.domainsWithPlansOnly }
-						onButtonClick={ this.addRemoveDomainToCart.bind( null, suggestion ) } />
+						onButtonClick={ this.handleClickResult.bind( null, suggestion ) } />
```

Comment:
I'm getting an `Uncaught TypeError: Cannot read property 'bind' of undefined`  error here when trying to navigate to `/domains/add/{site}` (either using the `Add Domain` button in Domain Management or the `Add` button in sidebar).